components/feature/customtabs/src/test/java/mozilla/components/feature/customtabs/CustomTabIntentProcessorTest.kt

```diff
@@ -150,6 +151,7 @@ class CustomTabIntentProcessorTest {
         val sessionCaptor = argumentCaptor<Session>()
         verify(sessionManager).add(sessionCaptor.capture(), eq(false), eq(null), eq(null), eq(null))
 
+        store.waitUntilIdle()
```

Comment:
@jonalmeida If we need this here, we should add it to the other tests as well before we call `assertFirstAction`?

Reply:
Since the follow the same pattern `.process(intent)`, `sessionManager.add()`, then `assertFirstAction`.

Reply:
Good catch! I was running the tests together without failure, but when I ran them individually I reached the same failure in all of them.